src/openrct2/ride/TrackDesign.h

```diff
@@ -78,7 +78,7 @@ struct TrackDesignMazeElement
 };
 
 class DataSerialiser;
-
+enum class RideMode : uint8_t;
```

Comment:
This seems to be a leftover from when you tried to change the TrackDesign type. I won't ask for removal, because I think you can make it work successfully now :)
```suggestion
enum class RideMode : uint8_t;

```

Reply:
After proper rebasing everything is ok :)